src/features/hotspots/details/HotspotDetailChart.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { ChartData } from '../../../components/BarChart/types'
+import Box from '../../../components/Box'
+import Text from '../../../components/Text'
+import ChartContainer from '../../../components/BarChart/ChartContainer'
+
+type Props = {
+  title: string
+  number?: string
+  change?: string
+  percentage?: number
+  data: ChartData[]
+  color: string
+}
+const HotspotDetailChart = ({
+  title,
+  number,
+  change,
+  percentage,
+  data,
+  color,
+}: Props) => {
+  const { t } = useTranslation()
+  const [focusedData, setFocusedData] = useState<ChartData | null>()
+  const onFocus = (chartData: ChartData | null) => {
+    setFocusedData(chartData)
+  }
+  return (
+    <Box marginBottom="m">
+      <Text fontSize={16} color="black" marginVertical="s">
```

Comment:
I think you need to use a variant here or it won't use the right font.
Likely something like
`<Text variant="body1" color="black" marginVertical="s">`